packages/compass/src/main/index.ts

```diff
@@ -24,7 +24,7 @@ initializeElectronRemote();
 installEarlyLoggingListener();
 installEarlyOpenUrlListener();
 
-process.title = `${app.getName()} ${app.getVersion()}`;
+process.title = app.getName();
```

Comment:
Why can't we just remove it? Shouldn't take the title from the Info.plist?

Reply:
https://github.com/mongodb-js/compass/commit/01838395c179e7ea9b9109ff771012d8c7e91c2c

`process.title` also overrides the actual command line, so that tools like `ps` don't display it (because the command line could include e.g. a connection string with credentials or a `--password` argument). We do the same in mongosh.